sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/MathExpressionsSuite.scala

```diff
@@ -760,4 +760,30 @@ class MathExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
       checkEvaluation(WidthBucket(Literal(v), Literal(s), Literal(e), Literal(n)), expected)
     }
   }
+
+  test("SPARK-37388: width_bucket") {
```

Comment:
shall we have a more descriptive name e.g. `SPARK-37388: width_bucket handles null input correctly`

Reply:
This test also tests other (edge) cases (normal inputs, NaN inputs, etc). There were no unit tests for `WidthBucket` with double inputs before, so I added some here.